.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,42 @@
-Resolves #
-
-Make sure you've covered the following bases in your PR. They won't all be
-needed for every PR (ex. fixing a straightforward rubocop issue might have no
-tradeoffs and add no dependencies).
-
-- [ ] Did you fill in the `Resolves #<issue number>` in the first line?
-- [ ] Is this a work in progress? If so, put "WIP" in the title.
-- [ ] What were the main changes/goals of this PR?
-- [ ] Did you need to make any tradeoffs? Why did you make the choice you made?
-- [ ] Did you add any dependencies?
-- [ ] Anything else we should know about?
+<!--Read comments, before commiting pull request read checklist again
+
+# Checklist:
+
+- I have performed a self-review of my own code
+- I have commented my code, particularly in hard-to-understand areas
+- I have made corresponding changes to the documentation
+- I have added tests that prove my fix is effective or that my feature works
+- New and existing unit tests pass locally with my changes
+-->
+
+# Resolves #001 <!--fill issue number-->
+
+# Description
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
+List any dependencies that are required for this change. Put "WIP" in the title if work is in progress. Did you add any dependencies?
+Include anything else we should know about. -->
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
Could you change these to either bullet points or checked checkboxes? I'm getting tired of "0 of X" on the PR page from my ill-conceived template.